modules/targets/generic-linux.nix

```
@@ -43,6 +43,18 @@ in {
       "/var/lib/snapd/desktop"
     ];
 
+    # We need to append system-wide FHS directories due to the default prefix
+    # resolving to the Nix store.
+    # https://github.com/nix-community/home-manager/pull/2891#issuecomment-1101064521
+    home.sessionVariables = {
+      XCURSOR_PATH = concatStringsSep ":" [
+        "\${XCURSOR_PATH}"
```

```suggestion
      XCURSOR_PATH =
        "$XCURSOR_PATH\${XCURSOR_PATH:+:}"
        + concatStringsSep ":" [
```
This will need to be formatted.

---

I forgot to consider the case where `XCURSOR_PATH` is null or unset. Thanks!